src/components/TokenStatsDisplay.tsx

```diff
@@ -0,0 +1,95 @@
+import clsx from "clsx";
+import Container from "@components/commons/Container";
+import TokenStatsApi from "@api/TokenStatsApi";
+import TrendLineChart, { LineData } from "./TrendLineChart";
+
+export interface TokenStats {
+  tokenPrice: number;
+  percentChange: number;
+  circulation: string;
+  last24hVolume: string;
+  marketCap: string;
+}
+
+const DmxTokenSymbol = "DMXTc";
+
+export default function TokenStatsDisplay(): JSX.Element {
+  const stats: TokenStats = TokenStatsApi.useTokenStats();
+  const priceHistory = TokenStatsApi.useTokenPriceHistory();
+
+  const trendLineData: LineData[] = priceHistory.map((data) => ({
+    value: data.price,
+  }));
+
+  return (
+    <Container className="px-5 sm:py-5 sm:px-10 md:py-6">
+      <section className="flex flex-col sm:flex-row sm:flex-wrap xl:flex-nowrap xl:items-center xl:justify-between p-5 pb-10 sm:p-10 xl:py-7 border-[0.5px] rounded-[15px]">
+        <section className="flex flex-1 xl:flex-1 xl:order-first xl:items-center">
+          <TokenPriceSection data={stats} />
+        </section>
+        <section className="xl:order-last xl:ml-16">
+          <DetailsSection data={stats} />
+        </section>
+        <section className="mt-8 sm:mt-12 xl:mt-0 h-[88px] sm:basis-full xl:order-2 xl:basis-auto xl:w-[304px]">
+          <TrendLineChart data={trendLineData} />
+        </section>
+      </section>
+    </Container>
+  );
+}
+
+function TokenPriceSection({ data }: { data: TokenStats }) {
+  return (
+    <>
+      <div className="h-10 w-10 sm:h-[39px] sm:w-[39px] xl:h-[46px] xl:w-[46px] sm:mt-0 rounded-full bg-gradient-to-r from-green-800 to-blue-800" />
```

Comment:
needs to vertically center this gradient circle in mobile too

Reply:
Updated. I also added a little margin for tablet UI